.env.testnet

```diff
@@ -13,3 +13,10 @@ NEXT_PUBLIC_GRANTS_ACTIVE_BUCKET_ADDRESS=0x2217E4d3Ae0A6E30075D1B5a7b8C1520E8009
 NEXT_PUBLIC_GROWTH_BUCKET_ADDRESS=0x420f39968009a0e8693a0e1bba633cd99eaffb9b
 NEXT_PUBLIC_GENERAL_BUCKET_ADDRESS=0x72Ed7d7b7835Ad62B1f9b6280bAd62618aA71461
 NEXT_PUBLIC_CHAIN_ID=31
+
+# CR-related env variables
+NEXT_PUBLIC_SIMPLIFIED_REWARD_DISTRIBUTOR_ADDRESS=0x4447EC87854651ECdce3E91e581c34641C91eC46
```

Comment:
These new variables wasn't added to all .env files which makes them useless.

Reply:
@rodrigoncalves good catch 👍🏼 
@jurajpiar it looks like we didn't update the files `.env.dev`, `.env.prod`, `.env.qa`.

Reply:
done, though I thought we weren't going to touch the orig dao configs.

Reply:
It's still missing some variables in orig dao configs, I can't run the code without them. Are they being replaced by the new ones? The thing is that it's still not clear to me how we are going to do for now on, specially for running locally.